apps/src/templates/teacherNavigation/TeacherNavigationRouter.tsx

```diff
@@ -236,8 +241,12 @@ const TeacherNavigationRouter: React.FC<TeacherNavigationRouterProps> = ({
             element={
               <ElementOrEmptyPage
                 showNoStudents={studentCount === 0}
+                showNoUnitAssigned={showNoUnitAssigned}
+                courseName={selectedSectionCourseName}
                 showNoCurriculumAssigned={!anyStudentHasProgress}
-                element={<LessonMaterialsContainer />}
+                element={applyV1TeacherDashboardWidth(
```

Comment:
I am wondering about wrapping the component inside this width.  Since `LessonMaterialsContainer` is a new component, I am wondering if this `V1` wrapper is needed.  Perhaps it makes centering the "EmptyElement" easier, but I wonder if we expect to have this `applyV1TeacherDashboardWidth` wrapper in for the long term (aka forever) or if this would go away once we roll the feature out.  @lfryemason

Reply:
Noting that I initially had wrapped it myself as Ken is doing here, but Eric had suggested defining the width within `LessonMaterialsContainer` because it is a new component.

Reply:
I agree that we shouldn't be using `applyV1TeacherDashboardWidth` for new components. This is just to make sure that existing components didn't break. Hopefully we are making new pages width responsive, but if not we should at least define the width in the component styles itself instead of using this v1 function